kubejs/server_scripts/gregtech/supercomputer.js

```diff
@@ -6,16 +6,35 @@ ServerEvents.recipes(event => {
         .itemInputs('gtceu:atomic_casing', '6x gcyr:trinaquadalloy_plate', '4x hostilenetworks:sim_chamber', 'kubejs:heart_of_a_universe', '2x gtceu:uv_robot_arm', '2x gtceu:uv_field_generator', '4x gtceu:wetware_processor_mainframe', 'draconicevolution:draconic_energy_core')
         .inputFluids('gtceu:soldering_alloy 1152')
         .itemOutputs('gtceu:simulation_supercomputer')
+        // requires research (sim chamber with data module)
         .duration(1200)
         .EUt(1966080)
         
     ////// Computer Recipes //////
 
-            // event.recipes.gtceu.simulation_supercomputer('kubejs:enderman')
-            //     .notConsumable(Item.of('hostilenetworks:data_model', '{data_model:{data:6,id:"hostilenetworks:enderman"}}'))
-            //     .itemInput('hostilenetworks:prediction_matrix')
-            //     .chancedOutput(Item.of('hostilenetworks:prediction', '{data_model:{id:"hostilenetworks:enderman"}}'), 5000, 500)
-            //     .Output(Item.of("hostilenetworks:prediction_enderman_prediction"))
-            //     .duration(512)
-            //     .EUt(10)
+    function supercomp(model, prediction, EUt) {
+        event.recipes.gtceu.simulation_supercomputer(`kubejs:${model}`)
+            .notConsumable(Item.of('hostilenetworks:data_model', '{data_model:{id:"hostilenetworks:' + model + '"}}')) // model nbts are still brokey
```

Comment:
```suggestion
            .notConsumable(Item.of('hostilenetworks:data_model', '{data_model:{id:"hostilenetworks:' + model + '"}}').weakNBT()) // model nbts are still brokey
```

Reply:
![image](https://github.com/ThePansmith/Monifactory/assets/75553966/aa7732c5-d136-42c6-9b2f-d173b847e412)
ITS PEAK, THANK YOU